app/views/admin/requests/index.html.haml

```diff
@@ -1,4 +1,4 @@
-%h1 Requests to Freelex
+%h1 Requests to Signbank
```

Comment:
We don't need this controller or template at all anymore, since we're not recording any requests.